src/collections/components/AuthorListCard/AuthorListCard.tsx

```diff
@@ -18,20 +18,33 @@ interface AuthorListCardProps {
  * @param props
  */
 export const AuthorListCard: React.FC<AuthorListCardProps> = (props) => {
-  const classes = useStyles();
   const { author } = props;
 
   // We pass the author object along with the link so that when the user clicks
   // on the card to go to an individual author's page, the page is loaded instantly.
   return (
     <Link
+      component={RouterLink}
       to={{
         pathname: `/collections/authors/${author.externalId}/`,
         state: { author },
       }}
-      className={classes.link}
+      sx={{
+        textDecoration: 'none',
+        padding: '1.25 rem 0',
+      }}
     >
-      <Card variant="outlined" square className={classes.root}>
+      <Card
+        square
+        sx={{
+          margin: '1rem auto',
+          padding: '1.25rem 0.25rem',
+          cursor: 'pointer',
+          '&:active': {
+            backgroundColor: curationPalette.neutral,
+          },
+        }}
+      >
```

Comment:
i see my previous question is answered here 😄. 

we don't need to be hung up on it during this upgrade, but is the above a good strategy for going forward? these are custom styles - but would they be better made into proper classes and put into the theme?

Reply:
Yeah I think we could benefit from custom css classes here. Although I do understand that might expand the scope a bit.

Reply:
Ideally having custom style variables works the best, that prevents us from hardcoding values. Like we have `curationPalette.neutral` but that requires defining a whole design system and we are not a front end team 😄